### PR TITLE
CNV-20936: Live YAML view RN

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.12

Issue: [CNV-20936](https://issues.redhat.com//browse/CNV-20936)

Link to docs preview: [RN: UI new features](https://54128--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-web-new)

QE review: @gouyang 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by gouyang** [comment]
LGTM

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -123,7 +123,7 @@ The SVVP Certification applies to:
 === Web console
 
 //CNV-20936 UI - live YAML view
-
+* When you customize a template to create a VM, you can set the *YAML* slider to *ON* on each VM configuration tab to view the live changes in the YAML configuration file alongside the form.
```
Comment (kcarmichael08) [inline virt/virt-4-12-release-notes.adoc:126]: I think normally if only "on/off" is provided, that element is called a switch, not a slider. See IBM SG: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#slider

Comment (kcarmichael08) [reply virt/virt-4-12-release-notes.adoc:126]: other suggestions:
- "on each VM configuration tab" - "in each VM configuration tab"
- alongside - next to

**Review by kcarmichael08 — COMMENTED** [review]
Couple of suggestions and comments; otherwise, lgtm!

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (apinnick) [reply virt/virt-4-12-release-notes.adoc:126]: IGS says the correct preposition for a tab is "on". https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tab

**Pull request merged**